draft-ietf-mls-protocol.md

```diff
@@ -967,6 +977,100 @@ update_secret -> HKDF-Extract = epoch_secret
                init_secret_[n]
 ~~~~~
 
+## Encryption Keys
+
+As described in {{message-framing}}, MLS encrypts two different
+types of information:
+
+* Handshake messages
+* Application messages
+
+Each handshake message is encrypted using a key and a nonce derived
+from the handshake_secret for a specific sender to prevent two senders
+to perform in the following way:
```

Comment:
This text assumes that there's only a need for one handshake message to be sent per epoch.  I realize that should be true in most cases, but I'm concerned about things like retries leading to nonce reuse.  This is why in #153, I had handshake keys follow the same path as application keys.  This also allows code reuse.

Reply:
Oh, I see you have conveniently not deleted that text :)  So I would just delete the text you've added up here.

Reply:
I can live with that for now.  Please remove the para starting "The same rules are used to generate a sequence of keys and nonces for handshake encryption"

I see what you're saying, I don't feel 100% confident that we won't get wedged by this in practice.  And I dislike the extra code paths.  But we can merge with this flavor and ask the WG.

Reply:
I agree, there might be a better solution but I don't immediately have it.
Should I add an open issue or something ?